src/main/java/org/folio/service/impl/TlrSettingsPublishCoordinatorServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package org.folio.service.impl;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.folio.client.feign.ConsortiaClient;
+import org.folio.domain.dto.PublicationRequest;
+import org.folio.domain.dto.PublicationResponse;
+import org.folio.domain.dto.Tenant;
+import org.folio.domain.dto.TlrSettings;
+import org.folio.domain.dto.UserTenant;
+import org.folio.service.PublishCoordinatorService;
+import org.folio.service.UserTenantsService;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class TlrSettingsPublishCoordinatorServiceImpl implements PublishCoordinatorService<TlrSettings> {
+  private static final String CIRCULATION_SETTINGS_URL = "/circulation/settings";
+  private static final String ECS_TLR_FEATURE = "ecsTlrFeature";
+  private final UserTenantsService userTenantsService;
+  private final ConsortiaClient consortiaClient;
+
+  @Override
+  public PublicationResponse updateForAllTenants(TlrSettings tlrSettings) {
+    log.debug("updateForAllTenants:: parameters: {} ", () -> tlrSettings);
+    UserTenant firstUserTenant = userTenantsService.findFirstUserTenant();
+    PublicationResponse publicationResponse = null;
+    if (firstUserTenant != null) {
```

Comment:
Can we flip the condition and make it into a guard clause to get rid of some nesting?
```
if (firstUserTenant != null) {
   log.error("updateForAllTenants:: userTenant was not found");
   return null;
}

// handle found "user tenant"

Reply:
done